src/main/kotlin/no/skatteetaten/aurora/boober/service/vault/EncryptedFileVault.kt

```diff
@@ -23,8 +23,7 @@ class VaultCollection private constructor(
         private val decryptor: Decryptor) {
 
     companion object {
-        fun fromFolder(folder: File, encryptor: Encryptor, decryptor: Decryptor): VaultCollection
-                = VaultCollection(folder, encryptor, decryptor)
+        fun fromFolder(folder: File, encryptor: Encryptor, decryptor: Decryptor): VaultCollection = VaultCollection(folder, encryptor, decryptor)
```

Comment:
what should our Kotlin code style say about line length?

Reply:
The java style guide and checkstyle would probably bork on this line length. I find it rather useful that these kinds of one line functions are on one line. 
What do you other people think?

Reply:
Most of the time I would prefer bodyless function implementations to on the same line as the signature, but I guess there is a limit to how long the line can be before it impedes readability. Actually, in this particular case, I'm torn. I'm fine with either.

Reply:
This style guide suggest a column limit of 100 chars: https://android.github.io/kotlin-guides/style.html

I don't mind 120 but longer lines than that quickly become distracting for me.

In either case we should decide on a standard.